packages/api/models/granules.js

```diff
@@ -650,7 +649,15 @@ class Granule extends Manager {
       updateParams.ConditionExpression += ' and #execution <> :execution';
     }
 
-    return this.dynamodbDocClient.update(updateParams).promise();
+    try {
+      return await this.dynamodbDocClient.update(updateParams).promise();
+    } catch (error) {
+      if (error.name && error.name.includes('ConditionalCheckFailedException')) {
```

Comment:
A delayed `running` event isn't the only possible cause of the conditional check exception though? It could also be because of the `createdAt` timestamp condition. I guess that only makes the log message invalid, but not the general logic?

Reply:
True. I can update the log message.

Reply:
https://github.com/nasa/cumulus/pull/2156/commits/1027bd6a4454d070ed340d950ee02e03372f90d2